Update multiplay.md

kyuu-ni-yon opened the pull request:
增加了部分mp指令。
然后把重要的!mp kick忘记了...

kyuu-ni-yon committed:
Update multiplay.md

增加了部分mp指令。

Evisolpxe reviewed (commented):
还可以补充几个常用的
!mp team 'username' 'color'
!mp move 'username' 'slot'
!mp timer 60

kyuu-ni-yon committed:
Update multiplay.md

增加了
!mp kick username
!mp team username color
!mp move username slot
!mp timer n
!mp invite username

kyuu-ni-yon commented:
@Evisolpxe 感谢提醒，已补充。
